autoload/miniSnip.vim

```diff
@@ -62,7 +62,13 @@ function! s:directories() abort
 endfunction
 
 function! s:findSnippetFile() abort
-  let cword = matchstr(getline('.'), '\v\f+%' . col('.') . 'c')
+  if exists("g:miniSnip_expandpattern")
+    let expandpattern = s:getVar("expandpattern")
+  else
+    let expandpattern = '\v\f+%'
+  endif
```

Comment:
It should be only `s:getVar("expandpattern")`. Setting default value should happen in `plugin/miniSnip.vim`